cypress/e2e/swap.test.ts

```diff
@@ -222,23 +224,33 @@ describe('Swap', () => {
 
             // Select USDC as output token from the common bases menu.
             cy.get('#swap-currency-output .open-currency-select-button').click()
-            cy.contains('USDC').click()
+            cy.get(getTestSelector('token-search-input')).clear().type('UNI')
+            cy.contains('UNI').click()
 
             // Set the input amount to the max amount minus the gas buffer.
-            cy.get('#swap-currency-input .token-amount-input').clear().type(swapInputAmount)
-            cy.get('#swap-currency-output .token-amount-input').should('not.equal', '')
-            cy.get('#swap-button').click()
-            cy.get('#confirm-swap-or-send').click()
-            cy.get(getTestSelector('dismiss-tx-confirmation')).click()
+            for (let i = 0; i < NUMBER_OF_SWAPS; i++) {
+              cy.get('#swap-currency-input .token-amount-input').clear().type(INDIVIDUAL_SWAP_INPUT.toString())
+              cy.get('#swap-currency-output .token-amount-input').should('not.equal', '')
+              cy.get('#swap-button').click()
+              cy.get('#confirm-swap-or-send').click()
+              cy.get(getTestSelector('dismiss-tx-confirmation')).click()
+            }
+
+            // The pending transaction indicator should be visible.
+            cy.contains(`${NUMBER_OF_SWAPS} Pending`).should('exist')
+
+            cy.then(() => hardhat.mine(1, 12)).then(() => {
```

Comment:
This is not testing timing, so you can just call `hardhat.mine()` without any args.